src/pages/ScreeningsPage.vue

```diff
@@ -1,11 +1,156 @@
 <template>
-  <div></div>
+  <div>
+    <h1>Screenings</h1>
+    <h2>{{ selectedDay }}</h2>
+    <div>
+      <button
+        class="day-tabs"
+        v-for="day in daysList"
+        :key="day.id"
+        :data-date="day.date.toJSON()"
+        @click="setSelectedDay"
+      >
+        {{ day.id === 1 ? day.name : niceDay(day.name) }}
+      </button>
+      <button class="day-tabs__calendar">
+        <img src="@/assets/img/calendaricon.svg" alt="calendar-icon" />
+      </button>
+      <hr />
+      <div v-for="movieId in moviesOnTheDay" :key="`movie_${movieId}`">
+        <p>
+          {{
+            $store.getters.movieList.filter((movie) => movie.id == movieId)[0]
+              .title
+          }}
+        </p>
+        <p
+          v-for="seance in seancesList
+            .filter(
+              (seance) => seance.datetime.substring(0, 10) === selectedDay
+            )
+            .filter((seance) => seance.movie == movieId)"
+          :key="`seance_${seance.id}`"
+        >
+          {{ seance.datetime.substring(11, 16) }}
+        </p>
+      </div>
+    </div>
+  </div>
 </template>
 
 <script>
+import apiSeancesService from "@/services/api/apiSeancesService";
+
 export default {
   name: "ScreeningsPage",
+  data() {
+    return {
+      daysList: [],
+      selectedDay: "",
+      seancesList: [],
+      seanceOnTheDay: [],
+      moviesOnTheDay: [],
+    };
+  },
+  computed: {
+    todayDate() {
+      return new Date();
+    },
+    todayDaySelection() {
+      let date = new Date();
+      return date.toJSON().substring(0, 10);
```

Comment:
[just an opinion] I think that `toISOString()` is a little bit more verbose than `toJSON()`, wdyt?